### PR TITLE
Add batch calls for faster deployment

### PR DESCRIPTION
**Pull request by Maar-io**
Replaced with #62

### CODE REVIEW

File: crates/multiasset/src/lib.rs [hunk 1]
```diff
@@ -129,6 +131,22 @@ where
         Ok(())
     }
 
+    /// Used to add an asset to a vector of tokens.
+    /// tokenId - ID of the token to add the asset to
+    /// assetId - ID of the asset to add to the token
```
Comment (Maar-io) [inline crates/multiasset/src/lib.rs:136]: fix comment. It is a vec now

**Comment by boyswan** [comment]
I wonder if these could be added as a separate `Batch` trait, as they're for a very specific use case will likely not need them.

Are we also sure these should be part of the library itself? It would be fairly straightforward for a user to implement these in their own contract if they required them, we could have an example contract to demonstrate. This way we wouldn't have to commit to arbitrary iteration limits, as well as maintaining functionality that is possibly a user concern rather than a library concern.

**Comment by Maar-io** [comment]
I agree with your comment that batch calls should not be in the library files.
I added a separate trait in `rmrk/batch.rs`
Please take a look if you had something like this in mind (code not complete)
https://github.com/rmrk-team/rmrk-ink/tree/feature/batch2

**Comment by boyswan** [comment]
> I agree with your comment that batch calls should not be in the library files. I added a separate trait in `rmrk/batch.rs` Please take a look if you had something like this in mind (code not complete) https://github.com/rmrk-team/rmrk-ink/tree/feature/batch2

Really like this, much cleaner!

**Pull request closed without merging**